client/components/marketing-survey/gsuite-cancel-purchase-dialog/gsuite-cancellation-survey.jsx

```diff
@@ -21,18 +22,18 @@ class GSuiteCancellationSurvey extends Component {
 		const {
 			disabled,
 			onSurveyAnswerChange,
+			purchase: { productSlug },
 			surveyAnswerId,
 			surveyAnswerText,
 			translate,
 		} = this.props;
+
 		return (
 			<MultipleChoiceQuestion
 				answers={ [
 					{
 						id: 'too-expensive',
 						answerText: translate( "It's too expensive." ),
-						textInput: true,
-						textInputPrompt: translate( 'How can we improve G Suite?' ),
```

Comment:
I think this prompt is misleading. First, it doesn't really relate to the question being asked. And second, we're not in a position of improving G Suite anyway (we're just a reseller of that product, we don't own it). I believe the following may be more appropriate (or something along that line):

> Which price per user/month would you find reasonable?

@wensco, @daledupreez: what do you think? Would you recommend a different question? For context, you can have a look to the results from the cancelation survey (a link can be found in the Asana card at `1199348849700520-as-1199353606986638`).

Reply:
I definitely agree that the current responses here are not useful, so I agree with removing or replacing the prompt.

Maybe phrasing it as "How much would you want to pay for %(product)s?" might get us better information. But I don't feel strongly about keeping the question.